blueprints/hr_assistant/entities/date_compare/mapping.json

```diff
@@ -0,0 +1,25 @@
+{
+    "entities": [
+        {
+            "whitelist": [
+                "before",
+                "pre",
+                "prior to"
+            ],
+            "cname": "prev"
+        },
+        {
+            "whitelist": [
+                "after",
+                "last",
+                "post",
+                "this year",
+                "this month",
+                "this week",
+                "atleast",
+                "since"
```

Comment:
How are `this year`, `this month`, and `this week` post date comparators? Should there be a third entry for something like `now`?

Reply:
for date_compare entities we first check if there is a certain time in the past (month, week, year etc.) being referred to. _post_ indicates if the user is referring to the duration _after_ the indicated time in the past and _prev_ means the user is referring to a time _before_ the indicated time in the past.

a phrase like "this month" would be "post" since it refers to the time from a month ago and onwards (to present)